README.md

```diff
@@ -48,10 +66,50 @@ jobs:
 
     # CLI for Microsoft 365 login action
     - name: Login to tenant
-      uses: pnp/action-cli-login@v2.0.0
+      uses: pnp/action-cli-login@v3.0.0
```

Comment:
The currently published action is v2.0.1. If we want to use a different version here, shouldn't we increase the version number in package.json?

Since we're adding an extra option to login without breaking the existing functionality, I'd suggest that we move to a v2.1.0 instead of v3.0.0.

Reply:
Oh I didn't see the version number on the package.json, I was only refering to the release notes in the README file.

The fact that all options are now optional (as they were required before) made me think about increasing major version. But I'm fine with increasing minor version.